tests/ballerina-unit-test/src/test/resources/test-src/jvm/anon-funcs.bal

```diff
@@ -69,3 +69,24 @@ function multilevelClosure() returns (function (int) returns int) {
     };
     return func1;
 }
+
+function basicWorkerTest() returns int {
```

Comment:
Can we add more tests? Or only the basic stuffs are working for workers?

Reply:
Worker interactions are not working yet. Without them I could not think of any complex scenarios. Any suggestions?